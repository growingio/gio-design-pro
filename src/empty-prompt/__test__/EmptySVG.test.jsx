import React from 'react';
import { render } from 'enzyme';
import EmptySVG from '../EmptySVG';

describe('EmptySVG', () => {
  it('render with default props', () => {
    const wrapper = render(<EmptySVG />);
    expect(wrapper).toMatchSnapshot();
  });
});
