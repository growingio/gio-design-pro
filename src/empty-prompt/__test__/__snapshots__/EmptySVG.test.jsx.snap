// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptySVG render with default props 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "height": "61",
      "viewBox": "0 0 60 61",
      "width": "60",
    },
    "children": Array [
      Object {
        "attribs": Object {
          "fill": "none",
          "fill-rule": "evenodd",
        },
        "children": Array [
          Object {
            "attribs": Object {
              "d": "M.588 36.882l10-21.764h38.824l10 21.764h-23.53a5.882 5.882 0 11-11.764 0H.588z",
              "fill": "#F7F8FC",
            },
            "children": Array [],
            "name": "path",
            "namespace": "http://www.w3.org/2000/svg",
            "next": Object {
              "attribs": Object {
                "d": "M.588 36.882h23.53a5.882 5.882 0 1011.764 0h23.53v20.53a3 3 0 01-3 3H3.588a3 3 0 01-3-3v-20.53z",
                "fill": "#FFF",
              },
              "children": Array [],
              "name": "path",
              "namespace": "http://www.w3.org/2000/svg",
              "next": Object {
                "attribs": Object {
                  "d": "M47.436 15.618a3.497 3.497 0 013.18 2.038h0l8.564 18.64c.211.458.32.957.32 1.461h0V57c0 .966-.392 1.841-1.025 2.475A3.489 3.489 0 0156 60.5h0H4a3.489 3.489 0 01-2.475-1.025A3.489 3.489 0 01.5 57h0V37.757a3.5 3.5 0 01.32-1.46h0l8.564-18.64a3.497 3.497 0 013.18-2.04h0z",
                  "stroke": "#DCDFED",
                },
                "children": Array [],
                "name": "path",
                "namespace": "http://www.w3.org/2000/svg",
                "next": Object {
                  "attribs": Object {
                    "d": "M.588 36.882h23.53a5.882 5.882 0 1011.764 0h23.53",
                    "stroke": "#DCDFED",
                    "stroke-linecap": "round",
                    "stroke-linejoin": "round",
                  },
                  "children": Array [],
                  "name": "path",
                  "namespace": "http://www.w3.org/2000/svg",
                  "next": Object {
                    "attribs": Object {
                      "d": "M30 1v9.412M40.588 2.261l-4.706 8.151M19.412 2.261l4.706 8.151",
                      "stroke": "#DCDFED",
                      "stroke-linecap": "round",
                      "stroke-linejoin": "round",
                      "stroke-width": "1.5",
                    },
                    "children": Array [],
                    "name": "path",
                    "namespace": "http://www.w3.org/2000/svg",
                    "next": null,
                    "parent": [Circular],
                    "prev": [Circular],
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "d": undefined,
                      "stroke": undefined,
                      "stroke-linecap": undefined,
                      "stroke-linejoin": undefined,
                      "stroke-width": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "d": undefined,
                      "stroke": undefined,
                      "stroke-linecap": undefined,
                      "stroke-linejoin": undefined,
                      "stroke-width": undefined,
                    },
                  },
                  "parent": [Circular],
                  "prev": [Circular],
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "d": undefined,
                    "stroke": undefined,
                    "stroke-linecap": undefined,
                    "stroke-linejoin": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "d": undefined,
                    "stroke": undefined,
                    "stroke-linecap": undefined,
                    "stroke-linejoin": undefined,
                  },
                },
                "parent": [Circular],
                "prev": [Circular],
                "type": "tag",
                "x-attribsNamespace": Object {
                  "d": undefined,
                  "stroke": undefined,
                },
                "x-attribsPrefix": Object {
                  "d": undefined,
                  "stroke": undefined,
                },
              },
              "parent": [Circular],
              "prev": [Circular],
              "type": "tag",
              "x-attribsNamespace": Object {
                "d": undefined,
                "fill": undefined,
              },
              "x-attribsPrefix": Object {
                "d": undefined,
                "fill": undefined,
              },
            },
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "d": undefined,
              "fill": undefined,
            },
            "x-attribsPrefix": Object {
              "d": undefined,
              "fill": undefined,
            },
          },
          Object {
            "attribs": Object {
              "d": "M.588 36.882h23.53a5.882 5.882 0 1011.764 0h23.53v20.53a3 3 0 01-3 3H3.588a3 3 0 01-3-3v-20.53z",
              "fill": "#FFF",
            },
            "children": Array [],
            "name": "path",
            "namespace": "http://www.w3.org/2000/svg",
            "next": Object {
              "attribs": Object {
                "d": "M47.436 15.618a3.497 3.497 0 013.18 2.038h0l8.564 18.64c.211.458.32.957.32 1.461h0V57c0 .966-.392 1.841-1.025 2.475A3.489 3.489 0 0156 60.5h0H4a3.489 3.489 0 01-2.475-1.025A3.489 3.489 0 01.5 57h0V37.757a3.5 3.5 0 01.32-1.46h0l8.564-18.64a3.497 3.497 0 013.18-2.04h0z",
                "stroke": "#DCDFED",
              },
              "children": Array [],
              "name": "path",
              "namespace": "http://www.w3.org/2000/svg",
              "next": Object {
                "attribs": Object {
                  "d": "M.588 36.882h23.53a5.882 5.882 0 1011.764 0h23.53",
                  "stroke": "#DCDFED",
                  "stroke-linecap": "round",
                  "stroke-linejoin": "round",
                },
                "children": Array [],
                "name": "path",
                "namespace": "http://www.w3.org/2000/svg",
                "next": Object {
                  "attribs": Object {
                    "d": "M30 1v9.412M40.588 2.261l-4.706 8.151M19.412 2.261l4.706 8.151",
                    "stroke": "#DCDFED",
                    "stroke-linecap": "round",
                    "stroke-linejoin": "round",
                    "stroke-width": "1.5",
                  },
                  "children": Array [],
                  "name": "path",
                  "namespace": "http://www.w3.org/2000/svg",
                  "next": null,
                  "parent": [Circular],
                  "prev": [Circular],
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "d": undefined,
                    "stroke": undefined,
                    "stroke-linecap": undefined,
                    "stroke-linejoin": undefined,
                    "stroke-width": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "d": undefined,
                    "stroke": undefined,
                    "stroke-linecap": undefined,
                    "stroke-linejoin": undefined,
                    "stroke-width": undefined,
                  },
                },
                "parent": [Circular],
                "prev": [Circular],
                "type": "tag",
                "x-attribsNamespace": Object {
                  "d": undefined,
                  "stroke": undefined,
                  "stroke-linecap": undefined,
                  "stroke-linejoin": undefined,
                },
                "x-attribsPrefix": Object {
                  "d": undefined,
                  "stroke": undefined,
                  "stroke-linecap": undefined,
                  "stroke-linejoin": undefined,
                },
              },
              "parent": [Circular],
              "prev": [Circular],
              "type": "tag",
              "x-attribsNamespace": Object {
                "d": undefined,
                "stroke": undefined,
              },
              "x-attribsPrefix": Object {
                "d": undefined,
                "stroke": undefined,
              },
            },
            "parent": [Circular],
            "prev": Object {
              "attribs": Object {
                "d": "M.588 36.882l10-21.764h38.824l10 21.764h-23.53a5.882 5.882 0 11-11.764 0H.588z",
                "fill": "#F7F8FC",
              },
              "children": Array [],
              "name": "path",
              "namespace": "http://www.w3.org/2000/svg",
              "next": [Circular],
              "parent": [Circular],
              "prev": null,
              "type": "tag",
              "x-attribsNamespace": Object {
                "d": undefined,
                "fill": undefined,
              },
              "x-attribsPrefix": Object {
                "d": undefined,
                "fill": undefined,
              },
            },
            "type": "tag",
            "x-attribsNamespace": Object {
              "d": undefined,
              "fill": undefined,
            },
            "x-attribsPrefix": Object {
              "d": undefined,
              "fill": undefined,
            },
          },
          Object {
            "attribs": Object {
              "d": "M47.436 15.618a3.497 3.497 0 013.18 2.038h0l8.564 18.64c.211.458.32.957.32 1.461h0V57c0 .966-.392 1.841-1.025 2.475A3.489 3.489 0 0156 60.5h0H4a3.489 3.489 0 01-2.475-1.025A3.489 3.489 0 01.5 57h0V37.757a3.5 3.5 0 01.32-1.46h0l8.564-18.64a3.497 3.497 0 013.18-2.04h0z",
              "stroke": "#DCDFED",
            },
            "children": Array [],
            "name": "path",
            "namespace": "http://www.w3.org/2000/svg",
            "next": Object {
              "attribs": Object {
                "d": "M.588 36.882h23.53a5.882 5.882 0 1011.764 0h23.53",
                "stroke": "#DCDFED",
                "stroke-linecap": "round",
                "stroke-linejoin": "round",
              },
              "children": Array [],
              "name": "path",
              "namespace": "http://www.w3.org/2000/svg",
              "next": Object {
                "attribs": Object {
                  "d": "M30 1v9.412M40.588 2.261l-4.706 8.151M19.412 2.261l4.706 8.151",
                  "stroke": "#DCDFED",
                  "stroke-linecap": "round",
                  "stroke-linejoin": "round",
                  "stroke-width": "1.5",
                },
                "children": Array [],
                "name": "path",
                "namespace": "http://www.w3.org/2000/svg",
                "next": null,
                "parent": [Circular],
                "prev": [Circular],
                "type": "tag",
                "x-attribsNamespace": Object {
                  "d": undefined,
                  "stroke": undefined,
                  "stroke-linecap": undefined,
                  "stroke-linejoin": undefined,
                  "stroke-width": undefined,
                },
                "x-attribsPrefix": Object {
                  "d": undefined,
                  "stroke": undefined,
                  "stroke-linecap": undefined,
                  "stroke-linejoin": undefined,
                  "stroke-width": undefined,
                },
              },
              "parent": [Circular],
              "prev": [Circular],
              "type": "tag",
              "x-attribsNamespace": Object {
                "d": undefined,
                "stroke": undefined,
                "stroke-linecap": undefined,
                "stroke-linejoin": undefined,
              },
              "x-attribsPrefix": Object {
                "d": undefined,
                "stroke": undefined,
                "stroke-linecap": undefined,
                "stroke-linejoin": undefined,
              },
            },
            "parent": [Circular],
            "prev": Object {
              "attribs": Object {
                "d": "M.588 36.882h23.53a5.882 5.882 0 1011.764 0h23.53v20.53a3 3 0 01-3 3H3.588a3 3 0 01-3-3v-20.53z",
                "fill": "#FFF",
              },
              "children": Array [],
              "name": "path",
              "namespace": "http://www.w3.org/2000/svg",
              "next": [Circular],
              "parent": [Circular],
              "prev": Object {
                "attribs": Object {
                  "d": "M.588 36.882l10-21.764h38.824l10 21.764h-23.53a5.882 5.882 0 11-11.764 0H.588z",
                  "fill": "#F7F8FC",
                },
                "children": Array [],
                "name": "path",
                "namespace": "http://www.w3.org/2000/svg",
                "next": [Circular],
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "d": undefined,
                  "fill": undefined,
                },
                "x-attribsPrefix": Object {
                  "d": undefined,
                  "fill": undefined,
                },
              },
              "type": "tag",
              "x-attribsNamespace": Object {
                "d": undefined,
                "fill": undefined,
              },
              "x-attribsPrefix": Object {
                "d": undefined,
                "fill": undefined,
              },
            },
            "type": "tag",
            "x-attribsNamespace": Object {
              "d": undefined,
              "stroke": undefined,
            },
            "x-attribsPrefix": Object {
              "d": undefined,
              "stroke": undefined,
            },
          },
          Object {
            "attribs": Object {
              "d": "M.588 36.882h23.53a5.882 5.882 0 1011.764 0h23.53",
              "stroke": "#DCDFED",
              "stroke-linecap": "round",
              "stroke-linejoin": "round",
            },
            "children": Array [],
            "name": "path",
            "namespace": "http://www.w3.org/2000/svg",
            "next": Object {
              "attribs": Object {
                "d": "M30 1v9.412M40.588 2.261l-4.706 8.151M19.412 2.261l4.706 8.151",
                "stroke": "#DCDFED",
                "stroke-linecap": "round",
                "stroke-linejoin": "round",
                "stroke-width": "1.5",
              },
              "children": Array [],
              "name": "path",
              "namespace": "http://www.w3.org/2000/svg",
              "next": null,
              "parent": [Circular],
              "prev": [Circular],
              "type": "tag",
              "x-attribsNamespace": Object {
                "d": undefined,
                "stroke": undefined,
                "stroke-linecap": undefined,
                "stroke-linejoin": undefined,
                "stroke-width": undefined,
              },
              "x-attribsPrefix": Object {
                "d": undefined,
                "stroke": undefined,
                "stroke-linecap": undefined,
                "stroke-linejoin": undefined,
                "stroke-width": undefined,
              },
            },
            "parent": [Circular],
            "prev": Object {
              "attribs": Object {
                "d": "M47.436 15.618a3.497 3.497 0 013.18 2.038h0l8.564 18.64c.211.458.32.957.32 1.461h0V57c0 .966-.392 1.841-1.025 2.475A3.489 3.489 0 0156 60.5h0H4a3.489 3.489 0 01-2.475-1.025A3.489 3.489 0 01.5 57h0V37.757a3.5 3.5 0 01.32-1.46h0l8.564-18.64a3.497 3.497 0 013.18-2.04h0z",
                "stroke": "#DCDFED",
              },
              "children": Array [],
              "name": "path",
              "namespace": "http://www.w3.org/2000/svg",
              "next": [Circular],
              "parent": [Circular],
              "prev": Object {
                "attribs": Object {
                  "d": "M.588 36.882h23.53a5.882 5.882 0 1011.764 0h23.53v20.53a3 3 0 01-3 3H3.588a3 3 0 01-3-3v-20.53z",
                  "fill": "#FFF",
                },
                "children": Array [],
                "name": "path",
                "namespace": "http://www.w3.org/2000/svg",
                "next": [Circular],
                "parent": [Circular],
                "prev": Object {
                  "attribs": Object {
                    "d": "M.588 36.882l10-21.764h38.824l10 21.764h-23.53a5.882 5.882 0 11-11.764 0H.588z",
                    "fill": "#F7F8FC",
                  },
                  "children": Array [],
                  "name": "path",
                  "namespace": "http://www.w3.org/2000/svg",
                  "next": [Circular],
                  "parent": [Circular],
                  "prev": null,
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "d": undefined,
                    "fill": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "d": undefined,
                    "fill": undefined,
                  },
                },
                "type": "tag",
                "x-attribsNamespace": Object {
                  "d": undefined,
                  "fill": undefined,
                },
                "x-attribsPrefix": Object {
                  "d": undefined,
                  "fill": undefined,
                },
              },
              "type": "tag",
              "x-attribsNamespace": Object {
                "d": undefined,
                "stroke": undefined,
              },
              "x-attribsPrefix": Object {
                "d": undefined,
                "stroke": undefined,
              },
            },
            "type": "tag",
            "x-attribsNamespace": Object {
              "d": undefined,
              "stroke": undefined,
              "stroke-linecap": undefined,
              "stroke-linejoin": undefined,
            },
            "x-attribsPrefix": Object {
              "d": undefined,
              "stroke": undefined,
              "stroke-linecap": undefined,
              "stroke-linejoin": undefined,
            },
          },
          Object {
            "attribs": Object {
              "d": "M30 1v9.412M40.588 2.261l-4.706 8.151M19.412 2.261l4.706 8.151",
              "stroke": "#DCDFED",
              "stroke-linecap": "round",
              "stroke-linejoin": "round",
              "stroke-width": "1.5",
            },
            "children": Array [],
            "name": "path",
            "namespace": "http://www.w3.org/2000/svg",
            "next": null,
            "parent": [Circular],
            "prev": Object {
              "attribs": Object {
                "d": "M.588 36.882h23.53a5.882 5.882 0 1011.764 0h23.53",
                "stroke": "#DCDFED",
                "stroke-linecap": "round",
                "stroke-linejoin": "round",
              },
              "children": Array [],
              "name": "path",
              "namespace": "http://www.w3.org/2000/svg",
              "next": [Circular],
              "parent": [Circular],
              "prev": Object {
                "attribs": Object {
                  "d": "M47.436 15.618a3.497 3.497 0 013.18 2.038h0l8.564 18.64c.211.458.32.957.32 1.461h0V57c0 .966-.392 1.841-1.025 2.475A3.489 3.489 0 0156 60.5h0H4a3.489 3.489 0 01-2.475-1.025A3.489 3.489 0 01.5 57h0V37.757a3.5 3.5 0 01.32-1.46h0l8.564-18.64a3.497 3.497 0 013.18-2.04h0z",
                  "stroke": "#DCDFED",
                },
                "children": Array [],
                "name": "path",
                "namespace": "http://www.w3.org/2000/svg",
                "next": [Circular],
                "parent": [Circular],
                "prev": Object {
                  "attribs": Object {
                    "d": "M.588 36.882h23.53a5.882 5.882 0 1011.764 0h23.53v20.53a3 3 0 01-3 3H3.588a3 3 0 01-3-3v-20.53z",
                    "fill": "#FFF",
                  },
                  "children": Array [],
                  "name": "path",
                  "namespace": "http://www.w3.org/2000/svg",
                  "next": [Circular],
                  "parent": [Circular],
                  "prev": Object {
                    "attribs": Object {
                      "d": "M.588 36.882l10-21.764h38.824l10 21.764h-23.53a5.882 5.882 0 11-11.764 0H.588z",
                      "fill": "#F7F8FC",
                    },
                    "children": Array [],
                    "name": "path",
                    "namespace": "http://www.w3.org/2000/svg",
                    "next": [Circular],
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "d": undefined,
                      "fill": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "d": undefined,
                      "fill": undefined,
                    },
                  },
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "d": undefined,
                    "fill": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "d": undefined,
                    "fill": undefined,
                  },
                },
                "type": "tag",
                "x-attribsNamespace": Object {
                  "d": undefined,
                  "stroke": undefined,
                },
                "x-attribsPrefix": Object {
                  "d": undefined,
                  "stroke": undefined,
                },
              },
              "type": "tag",
              "x-attribsNamespace": Object {
                "d": undefined,
                "stroke": undefined,
                "stroke-linecap": undefined,
                "stroke-linejoin": undefined,
              },
              "x-attribsPrefix": Object {
                "d": undefined,
                "stroke": undefined,
                "stroke-linecap": undefined,
                "stroke-linejoin": undefined,
              },
            },
            "type": "tag",
            "x-attribsNamespace": Object {
              "d": undefined,
              "stroke": undefined,
              "stroke-linecap": undefined,
              "stroke-linejoin": undefined,
              "stroke-width": undefined,
            },
            "x-attribsPrefix": Object {
              "d": undefined,
              "stroke": undefined,
              "stroke-linecap": undefined,
              "stroke-linejoin": undefined,
              "stroke-width": undefined,
            },
          },
        ],
        "name": "g",
        "namespace": "http://www.w3.org/2000/svg",
        "next": null,
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "fill": undefined,
          "fill-rule": undefined,
        },
        "x-attribsPrefix": Object {
          "fill": undefined,
          "fill-rule": undefined,
        },
      },
    ],
    "name": "svg",
    "namespace": "http://www.w3.org/2000/svg",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "height": undefined,
      "viewBox": undefined,
      "width": undefined,
    },
    "x-attribsPrefix": Object {
      "height": undefined,
      "viewBox": undefined,
      "width": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
